pinot-server/src/main/java/org/apache/pinot/server/api/resources/HealthCheckResource.java

```diff
@@ -90,11 +95,13 @@ public String checkReadiness() {
     return getReadinessStatus(_instanceId);
   }
 
-  private static String getReadinessStatus(String instanceId) throws WebApplicationException {
+  private String getReadinessStatus(String instanceId) throws WebApplicationException {
```

Comment:
(minor) No need to pass in the member variable
```suggestion
  private String getReadinessStatus() throws WebApplicationException {
```

Reply:
Thanks, done! Seems logical to not pass it since I'm making method non static.